CONTRIBUTING.md

```diff
@@ -0,0 +1,172 @@
+
+* A short, descriptive title. Ideally, other community members should be able
+  to get a good idea of the feature just from reading the title.
+* A detailed description of the the proposed feature. Include justification for
+  why it should be added to the SDK, and possibly example code to illustrate
+  how it should work.
+* [Markdown][markdown] formatting as appropriate to make the request easier to
+  read.
+* If you intend to implement this feature, indicate that you'd like to the
+  issue to be assigned to you
+
+## Code Contributions
+Code contributions to the SDK are done through [Pull Requests][pull-requests].
+Please keep the following in mind when considering a code contribution:
+
+* The SDK is released under the [Apache 2.0 License][license].
+
+   Any code you submit will be released under this license. If you are
+   contributing a large/substantial feature, you may be asked to sign a
+   [Contributor License Agreement (CLA)][cla].
+* For anything but very small or quick changes, you should always start by
+  checking the [Issues][issues] page to see if the work is already being done
+  by another person.
+
+  If you're working on a bug fix, check to see if the bug has already been
+  reported. If it has but no one is assigned to it, ask one of the maintainers
+  to assign it to you before beginning work.  If you're confident the bug
+  hasn't been reported yet, create a new [Bug Report](#bug-reports) then ask to
+  be assigned to it.
+
+  If you are thinking about adding entirely new functionality, open a [Feature
+  Request](#feature-requests) or [ping][gitter] the maintainers to ask for
+  feedback first before beginning work; again this is to make sure that no one
+  else is already working on it, and also that it makes sense to be included in
+  the SDK.
+* All code contributions must be accompanied with new or modified tests that
+  verify that the code works as expected; i.e. that the issue has been fixed or
+  that the functionality works as intended.
+
+### Your First Code Change
+For detailed information on getting started building and making code changes to
+the SDK, refer to our [Working on the SDK][working-on-the-sdk] wiki page.
+
+### Pull Request Readiness
+Before submitting your pull request, refer to the pull request readiness
+checklist below:
+
+* [ ] Includes tests to exercise the new behavior
+* [ ] Code is documented, especially public and user-facing constructs
+* [ ] Local run of `mvn install` succeeds
+* [ ] Git commit message is detailed and includes context behind the change
+* [ ] If the change is related to an existing Bug Report or Feature Request,
+  the issue number is referenced
+* [ ] A short description of the change added to
+  [CHANGELOG.md](./CHANGELOG.md). Adding a new entry can be accomplished by
+  running the `scripts/new-change` script and following the instructions.
+  Commit the new file created by the script in `.changes/next-release` with
+  your changes.
```

Comment:
This script isn't available for 1.11. The SDK team member that accepts your pull request will be able to add a changelog entry, though.

Reply:
Understood. Will remove item from list then.